### PR TITLE
fix documents about nativeWebTap

### PR DESCRIPTION
**Pull request by takeyaqa**
## Proposed changes

Since appium 1.7.0, cap `nativeWebTap`

  * can use setting api. appium/appium-xcuitest-driver#455
  * works on real device. appium/appium-xcuitest-driver#458

But documents have not been changed.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/advanced-concepts/settings.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ Note that the actual commands you would use in your test script differ based on
 
 |Name|Description|Values|Appium versions|
 |----|----|----|----|
+|`nativeWebTap`| Enable "real", non-javascript-based web taps in Safari. Default: `false`. Warning: depending on viewport size/ratio this might not accurately tap an element. | `true`, `false` |1.7.0+|
```
Comment (jlipps) [inline docs/en/advanced-concepts/settings.md]: `viewport size/ratio; this`

Comment (jlipps) [reply docs/en/advanced-concepts/settings.md]: (same below)

**Review by jlipps — APPROVED** [review]

File: docs/en/advanced-concepts/settings.md (hunk 1 above)
Comment (takeya0x86) [reply docs/en/advanced-concepts/settings.md]: Okey!

**Comment by jlipps** [comment]
thanks!

**Pull request merged**